arviz/data/datasets.py

```diff
@@ -213,8 +215,14 @@ def load_arviz_data(dataset=None, data_home=None):
         remote = REMOTE_DATASETS[dataset]
         home_dir = get_data_home(data_home=data_home)
         file_path = os.path.join(home_dir, remote.filename)
+
         if not os.path.exists(file_path):
-            urlretrieve(remote.url, file_path)
+            http_type = RcParams["data.http_protocol"]
+
+            # Replaces http type. Redundant if http_type is http, useful if http_type is https
+            url = remote.url.replace("http", http_type)
```

Comment:
@OriolAbril Let me know if this string replace method is fine by you. I can always do the string substitution method as well. Picked current pattern so urls would be valid if anyone wanted to copy and paste but I'm not tied to the pattern